src/evaluators/response/PHAL_ResponseSquaredL2DifferenceSide_Def.hpp

```diff
@@ -17,9 +17,24 @@ template<typename EvalT, typename Traits, typename SourceScalarT, typename Targe
 PHAL::ResponseSquaredL2DifferenceSideBase<EvalT, Traits, SourceScalarT, TargetScalarT>::
 ResponseSquaredL2DifferenceSideBase(Teuchos::ParameterList& p, const Teuchos::RCP<Albany::Layouts>& dl)
 {
-  // get response parameter list
+  // Get response parameter list
   Teuchos::ParameterList* plist = p.get<Teuchos::ParameterList*>("Parameter List");
 
+  // Validate response parameter list
```

Comment:
This is a good idea. We've all been bitten by params not correctly forwarded to the evaluator. Validation is a good idea, and  we should try to propagate it to other evals (if we have time), especially those that are auto-constructed, like responses.

Reply:
agreed, and it doesn't take that long to create the validation list.